osidb/api_views.py

```diff
@@ -345,6 +345,43 @@ def include_exclude_fields_extend_schema_view(
                 location=OpenApiParameter.QUERY,
                 deprecated=True,
             ),
+            OpenApiParameter(
+                "cvss2",
+                type=OpenApiTypes.STR,
+                location=OpenApiParameter.QUERY,
+                deprecated=True,
+            ),
+            OpenApiParameter(
+                "cvss2_score",
+                type=OpenApiTypes.FLOAT,
+                location=OpenApiParameter.QUERY,
+                deprecated=True,
+            ),
+            OpenApiParameter(
+                "cvss3",
+                type=OpenApiTypes.STR,
+                location=OpenApiParameter.QUERY,
+                deprecated=True,
+            ),
+            OpenApiParameter(
+                "cvss3_score",
+                type=OpenApiTypes.FLOAT,
+                location=OpenApiParameter.QUERY,
+                deprecated=True,
+            ),
+            OpenApiParameter(
+                "nvd_cvss2",
+                type=OpenApiTypes.STR,
+                location=OpenApiParameter.QUERY,
+                deprecated=True,
+            ),
+            OpenApiParameter(
+                "nvd_cvss3",
+                type=OpenApiTypes.STR,
+                location=OpenApiParameter.QUERY,
+                deprecated=True,
+            ),
+            # todo: cvss3 comment z meta attr?
```

Comment:
I think that there is no need to reflect the meta attribute changes in the API docs. There are no guarantees regarding the metadata.

Reply:
This does not belong here. It is just my old note.